tests/reporter/kubernetes.go

```diff
@@ -302,6 +305,63 @@ func (r *KubernetesReporter) logAuditLogs(virtCli kubecli.KubevirtClient, since
 	}
 }
 
+func (r *KubernetesReporter) logJournal(virtCli kubecli.KubevirtClient, duration time.Duration, unit string) {
+
+	component := "journal"
+	if unit != "" {
+		component += "_" + unit
+	}
+
+	logsdir := filepath.Join(r.artifactsDir, "nodes")
+	if err := os.MkdirAll(logsdir, 0777); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to create directory %s: %v\n", logsdir, err)
+		return
+	}
+
+	logDuration := strconv.FormatInt(int64(duration/time.Second), 10)
+
+	nodes := getNodesWithVirtLauncher(virtCli)
+
+	for _, node := range nodes {
+		pod, err := kubecli.NewVirtHandlerClient(virtCli).Namespace(tests.KubeVirtInstallNamespace).ForNode(node).Pod()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "failed to get virt-handler pod on node %s: %v", node, err)
+			continue
+		}
+
+		commands := []string{
+			"/usr/bin/virt-chroot",
+			"--mount",
+			"/proc/1/ns/mnt",
+			"exec",
+			"--",
+			"/usr/bin/journalctl",
+			"--since",
+			"-" + logDuration + "s",
+		}
+		if unit != "" {
```

Comment:
instead of two if, maybe worth to set a command_postfix that would always be appended, and would be set for -u unit in or empty string, in the first if ?

Reply:
Done